requirements/base.txt

```diff
@@ -6,7 +6,7 @@ aiohttp==3.8.4
 azure-core==1.26.3
 azure-storage-blob==12.14.1
 # upgrade setup.py after pinning certtifi to new version
-certifi==2022.12.7
+certifi==2023.5.7
```

Comment:
do we need this update? might cause some conflicts, but not sure

Reply:
You are right, lets skip it for now and address it in the future update